packages/dashmate/configs/getConfigFileMigrationsFactory.js

```diff
@@ -765,6 +765,15 @@ export default function getConfigFileMigrationsFactory(homeDir, defaultConfigs)
           });
         return configFile;
       },
+      '1.1.0-dev.1': (configFile) => {
+        Object.entries(configFile.configs)
+          .forEach(([, options]) => {
+            options.platform.drive.tenderdash.genesis
+              .consensus_params = base.get('platform.drive.tenderdash.genesis.consensus_params');
+            options.platform.drive.tenderdash.docker.image = base.get('platform.drive.tenderdash.docker.image');
```

Comment:
please also set max connections (to discuss):

```
max-connections = 64
max-outgoing-connections = 32
```

Reply:
Let's do it in a separate PR